src/modifiers/__test__/handleInlineStyle-test.js

```diff
@@ -151,7 +151,52 @@ describe("handleInlineStyle", () => {
         hasFocus: true,
       }),
     },
-    "converts semicolons to bold with astarisks": {
+    "converts to bold with asterisks on non-space character": {
+      before: {
+        entityMap: {},
+        blocks: [
+          {
+            key: "item1",
+            text: "hello **inline** style",
+            type: "unstyled",
+            depth: 0,
+            inlineStyleRanges: [],
+            entityRanges: [],
+            data: {},
+          },
+        ],
+      },
+      after: {
+        entityMap: {},
+        blocks: [
+          {
+            key: "item1",
+            text: "hello inlinea style",
```

Comment:
You probably mean "inline" here :)

Reply:
Actually, I didn't! This is how this test works:

Text before: (`|` is the cursor position)

```
hello inline| style
```

Then I press the "a" button, what do you expect to see? Exactly:

```
hello inlinea| style
```

Reply:
oh! sorry. well I've just learned something new. 🌟